src/cargo-fmt/main.rs

```diff
@@ -90,7 +94,27 @@ fn execute() -> i32 {
 
     let strategy = CargoFmtStrategy::from_opts(&opts);
 
-    handle_command_status(format_crate(verbosity, &strategy, opts.rustfmt_options))
+    if opts.manifest_path.is_some() {
+        let specified_manifest_path = opts.manifest_path.unwrap();
+        if !specified_manifest_path.ends_with("Cargo.toml") {
+            print_usage_to_stderr("the manifest-path must be a path to a Cargo.toml file");
```

Comment:
This is the same error message `cargo test` displays when the value is not a path to a Cargo.toml file, so I thought it would be good to do the same here.